### PR TITLE
Remove Ostatic link as it has been broken for a few weeks

### PR DESCRIPTION
**Pull request by sb2nov**
R: @davorbonaci PTAL

### CODE REVIEW

File: src/_posts/2017-02-01-graduation-media-recap.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ include:
 * Datanami: "[Google Lauds Outside Influence on Apache Beam](https://www.datanami.com/2017/01/10/google-lauds-outside-influence-apache-beam/)" by Alex Woodie.
 * InfoWorld / JavaWorld: "[Apache Beam unifies batch and streaming for big data](http://www.infoworld.com/article/3156598/big-data/apache-beam-unifies-batch-and-streaming-for-big-data.html)" by Serdar Yegulalp, and republished in [JavaWorld](http://www.javaworld.com/article/3156598/big-data/apache-beam-unifies-batch-and-streaming-for-big-data.html).
 * JAXenter: "[In a way, Apache Beam is the glue that connects many big data systems together](https://jaxenter.com/apache-beam-interview-131314.html)" by Kypriani Sinaris.
-* OStatic: "[Apache Beam Unifies Batch and Streaming Data Processing](http://ostatic.com/blog/apache-beam-unifies-batch-and-streaming-data-processing)" by Sam Dean.
```
Comment (davorbonaci) [inline src/_posts/2017-02-01-graduation-media-recap.md]: Just move the link into the comment; don't change the content otherwise.

Comment (sb2nov) [inline src/_posts/2017-02-01-graduation-media-recap.md]: Done

**Comment by sb2nov** [comment]
@davorbonaci PTAL again

**Comment by sb2nov** [comment]
@davorbonaci PTAL again

File: src/_posts/2017-02-01-graduation-media-recap.md (hunk 1 above)
Comment (davorbonaci) [inline src/_posts/2017-02-01-graduation-media-recap.md]: Same comment as before -- I'd prefer if the content of the blog doesn't change; just comment out the link, not the entire bullet.

**Comment by davorbonaci** [comment]
(commented in the discussion above)

File: src/_posts/2017-02-01-graduation-media-recap.md (hunk 1 above)
Comment (sb2nov) [inline src/_posts/2017-02-01-graduation-media-recap.md]: Sorry I didn't understand the comment correctly. Will fix.

Comment (sb2nov) [inline src/_posts/2017-02-01-graduation-media-recap.md]: Done

**Pull request closed without merging**